Userland/Libraries/LibGfx/CMYKBitmap.cpp

```diff
@@ -15,4 +15,21 @@ ErrorOr<NonnullRefPtr<CMYKBitmap>> CMYKBitmap::create_with_size(IntSize const& s
     return adopt_ref(*new CMYKBitmap(size, move(data)));
 }
 
+ErrorOr<RefPtr<Bitmap>> CMYKBitmap::to_rgb() const
```

Comment:
Can we name this `to_low_quality_rgb()` or similar? The main motivation for this class is to enable using ICC profiles for higher-fidelity CMYK->RGB conversion, so it'd be cool if we could discourage this about-to-be-fallback method :)

Personally, I'd also move the JPEG loader to this in the same commit and call this "Extract CMYK->RGB code from jpeg loader" or some such (but up to you).